addons/hw_posbox_homepage/controllers/main.py

```diff
@@ -104,8 +104,51 @@ def index(self):
         else:
             return homepage_template.render(self.get_homepage_data())
 
-    @http.route('/list_handlers', type='http', auth='none', website=True)
-    def list_handlers(self):
+    @http.route('/list_handlers', type='http', auth='none', methods=['GET', 'POST'], website=True, csrf=False)
```

Comment:
@mart-e do you have more details as of why it would risky ?

Reply:
the reason was convenience, as I send a form from the view (with a POST request), I can set another URL but then it would also redirect to this URL (except if I use jquery/Ajax or things like this).

Reply:
@lse-odoo for instance, in this code the `need_config_save` is only relevant in the scenario of a POST and yet it's partially present in the GET part too.

Reply:
@mart-e 
I don't understand what you mean by "partially present". You mean the case on which it is a post request it will persist after the `if request.httprequest.method == 'POST':` as in python variables are not deleted when exiting their definition scope ?

What would you think would be best then ? Have another controller to handle the POST request, in this case I would like that the form redirect to the "/list/list_handlers" page on form submission, is there a convenient way to do so from the page (apart from Jquery/AJAX calls) ?